cmd/eksctl/create.go

```diff
@@ -64,6 +64,7 @@ func createClusterCmd() *cobra.Command {
 
 	fs.StringVarP(&cfg.ClusterName, "cluster-name", "n", "", fmt.Sprintf("EKS cluster name (generated if unspecified, e.g. %q)", exampleClusterName))
 	fs.StringVarP(&cfg.Region, "region", "r", DEFAULT_EKS_REGION, "AWS region")
+	fs.StringVarP(&cfg.Profile, "profile", "p", "", "AWS Profile to use. This overrides the AWS_PROFILE environment variable if its set")
```

Comment:
Minor/Grammar: `if its set`. What about something like the below?
`AWS profile to use. If provided, this overrides the AWS_PROFILE environment variable`
N.B.: this comment applies to `delete.go`, `get.go` and `utils.go` as well.

Reply:
Yes i think that would be better. I'll make the change now.

Reply:
Thank you so much! :+1: